eggs/src/commands/init.ts

```diff
@@ -5,7 +5,15 @@ export const init = new Command()
     .version("0.1.0")
     .description("Initiates a new package for the nest.land registry.")
     .action(async () => {
-        if (pathExists("egg.json")) console.warn(yellow("An egg.json file already exists here! Overriding..."));
+        let previousConfig: Config = {}
```

Comment:
```suggestion
        let previousConfig: Config = {};
```

Maybe add a semicolon here :)

Reply:
Yep, sorry I didn't run `deno fmt` because it appears that it's not that way in every file
But I saw that on the next milestone there will be style guidelines so it shouldn't happen again :)